providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdProvider.java

```diff
@@ -630,4 +635,13 @@ public void restartEventStream() throws Exception {
 
         this.handleEvents();
     }
+
+    /**
+     * Call .wait() on this to block until the event stream is alive.
```

Comment:
Can you add a tiny bit of explanation here on why this might be needed? Maybe mention it's probably not need in prod. I just dont want people to think it's necessary when it's not.

Reply:
Done